README.md

```diff
@@ -34,13 +34,20 @@ import (
 
 	"github.com/scaleway/scaleway-sdk-go/api/instance/v1"
 	"github.com/scaleway/scaleway-sdk-go/scw"
+	"github.com/scaleway/scaleway-sdk-go/scwconfig"
 )
 
 func main() {
 
+	// Get Scaleway Config
+	config, err := scwconfig.Load() // Get your credentials at https://console.scaleway.com/account/credentials
```

Comment:
Keep the previous example in the readme, but add this in in [example_test.go](https://github.com/scaleway/scaleway-sdk-go/blob/master/example_test.go) 

```
func Example_apiClientWithConfig() {
	...
}
```